components/brave_savings/browser/BUILD.gn

```diff
@@ -0,0 +1,22 @@
+import("//brave/build/config.gni")
```

Comment:
`brave_savings` seems like an odd choice of naming to me since everything inside it is `perf_predictor`